4/bin/entrypoint.sh

```diff
@@ -1,7 +1,20 @@
 #!/bin/bash
 
-set -e
-set -o pipefail
+set -euo pipefail
+
+declare -a args
+
+add_env_var_as_env_prop() {
+  if [ ! -z "$1" ]; then
+    args+=("-D$2=$1")
+  fi
+}
+
+add_env_var_as_env_prop "${SONAR_HOST_URL:-}" "sonar.host.url"
```

Comment:
We can drop this as scanner-cli 4.2 natively supports `SONAR_HOST_URL` env variable